src/convertToObject.js

```diff
@@ -5,8 +5,21 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const arrStyles = sourceString.split(';').map((str) => str.trim());
```

Comment:
Consider adding a check to handle empty strings or strings without a colon, which could lead to unexpected results or errors.